### PR TITLE
Update worksheetprotection-unprotect.md

### PR DESCRIPTION
**Pull request by grangeryy**
Correct the wrong documentation on worksheetprotection. Excel Graph doesn't support password yet.

### CODE REVIEW

File: api-reference/beta/api/worksheetprotection-unprotect.md [hunk 1]
```diff
@@ -60,10 +56,8 @@ Here is an example of the request.
 ```http
 POST https://graph.microsoft.com/beta/me/drive/items/{id}/workbook/worksheets/{id|name}/protection/unprotect
 Content-type: application/json
-Content-length: 34
-
+workbook-session-id: {session-id}
 {
-  "password": "password-value"
```
Comment (Lauragra) [inline api-reference/beta/api/worksheetprotection-unprotect.md:66]: Do you still want to have an empty JSON body here? Also do we need the Content-type header? Note that it isn't listed in the request headers section.

File: api-reference/v1.0/api/worksheetprotection-unprotect.md [hunk 2]
```diff
@@ -58,10 +54,8 @@ Here is an example of the request.
 ```http
 POST https://graph.microsoft.com/v1.0/me/drive/items/{id}/workbook/worksheets/{id|name}/protection/unprotect
 Content-type: application/json
-Content-length: 34
-
+workbook-session-id: {session-id}
 {
-  "password": "password-value"
```
Comment (Lauragra) [inline api-reference/v1.0/api/worksheetprotection-unprotect.md:64]: The same questions apply to this topic re: the JSON body and content-type header.

File: api-reference/beta/api/worksheetprotection-unprotect.md (hunk 1 above)
Comment (grangeryy) [reply api-reference/beta/api/worksheetprotection-unprotect.md:66]: Thanks for pointing out. I'll remove the empty body. Also, no need for content-type header.

Comment (grangeryy) [reply api-reference/beta/api/worksheetprotection-unprotect.md:66]: sessionid header is optional.

**Pull request merged**